src/components/SignerPanel/Web3Errors.js

```diff
@@ -88,7 +99,16 @@ export const AccountLocked = ({
       actionText={
         <span>
           Please unlock and{' '}
-          <ButtonLink onClick={onRequestEnable}>enable</ButtonLink>{' '}
+          <ButtonText
+            onClick={onRequestEnable}
+            horizontalPadding="none"
+            css={`
+              padding: 0;
```

Comment:
…or maybe we need two components: `ButtonText` would be without any padding, horizontal nor vertical, and `Button` would have a `text` mode, with a way to disable the padding on one or both sides.